docs/design/3_2_multirotor.tex

```diff
@@ -118,103 +115,54 @@ \subsubsection{Air Frame}\label{section:air-frame}
 
 \paragraph{Motor Size and Speed}\label{section:motor-speed}
 
-All BLDC out-runner motors have the same height, and on product pages they are often given the code 22XX to denote their stator size. The design parameter is the radius (XX on the product code). Typically, the larger the stator (radius), the slower it spins given the same voltage. This is a linear relationship described by the motor parameter kV:
-
-$$
-\omega_{\mathrm{RPM}} = \mathrm{kV} \times \mathrm{Voltage}
-$$
-
-The kV constant is inversely proportional to another motor constant, kT, which models the linear relationship between current and torque:
-
-$$
-\tau = \mathrm{kT} \times \mathrm{Current}
-$$
+Commercial outrunner BLDC motors generally have the following design parameters: rotor size, KV or KT constants, voltage, maximum rated power, and weight. Please refer to Appendix~\ref{appendix:droneengine} for the relationship between the aforementioned design parameters. Our design process consists of screening and ranking the best motors using the following order:
 
-The inverse relationship can be shown by equating electrical power into the motor and mechanical power produced by the motor:
+\begin{enumerate}
+    \item Voltage: we choose motor that operates between 12.0 V to 20.0 V which is enough for heavy operations.
+    \item KV constant: Since the motors' RPM-per-volt ratio (KV constant) is inversely proportional of torque-per-current ratio (KT constant), manufacturers typically only focus on the KV constant. We choose motors that has KV constants between 500 KV to 1000 KV. These constants are considered low-speed and high-torque characteristics and are typically used for large drones and endurance flights. We also choose this range so the propellers can spin slower and more efficiently, which extends the maximum flight time.
```

Comment:
```suggestion
    \item KV constant: Since the motors' RPM-per-volt ratio (KV constant) is inversely proportional to the torque-per-current ratio (KT constant), manufacturers typically only focus on the KV constant. We choose motors that have KV constants between 500 KV to 1000 KV. These constants are considered low-speed and high-torque characteristics and are typically used for large drones and endurance flights. We also choose this range so the propellers can spin slower and more efficiently, which extends the maximum flight time.
```